README.md

```diff
@@ -42,6 +42,24 @@ Eyeball input & output files to ensure that the ClinVar format has not changed s
 enough to render this snippet invalid. Then put the generated files into
 `clinvar-xml-parser/src/test/resources/` directory.
 
+Deploying local OLS installation
+-------
+
+During the preparation of 2019_04 release, which had to be synchronized with EFO v3, OLS had to be deployed
+locally because the production deployment of OLS on www.ebi.ac.uk/ols only supported EFO v2 at the time.
+This can be done using the following command:
+
+```bash
+sudo docker run -p 8080:8080 simonjupp/efo3-ols:3.4.0
+```
+
+To use the local deployment, make note of the IP address of the machine where OLS is deployed. Then in
+`/eva_cttv_pipeline/trait_mapping/ols.py` uncomment the configuration section to specify the URL of the local
+installation (don't forget to substitute the correct IP address where the OLS installation is deployed).
+
+Contact Simon Jupp <jupp@ebi.ac.uk> or somebody from SPOT if you have questions about local OLS installation.
```

Comment:
The instruction is trivial for us, but it's better to have it included in case someone unfamiliar with the workflow will be running the code